cli/tools/lint.rs

```diff
@@ -406,6 +497,10 @@ impl LintReporter for JsonLintReporter {
     });
   }
 
+  fn reset_counter(&mut self) {
+    unimplemented!()
+  }
```

Comment:
This is strange... why is this function needed?

Reply:
https://github.com/denoland/deno/blob/4ce2c8b7b903f2ab4c883a85767d8f394aaf385a/cli/tools/lint.rs#L157

I need to call this function to reset the lint counter, otherwise, it will keep increasing between multiple runs of the operation

Reply:
Why not do it in `close()` instead?

Reply:
Or maybe I can create the report inside the operation closure every single time

Reply:
Also it seems that reporter could just be created on each restart of file watcher, that would allow to keep it "single use" and we won't need to worry about its internal state

Reply:
oh! my bad, just didn't think about it(didn't want to change existing behaviours, I will change it